grails-app/assets/stylesheets/CSS1/style.css

```
@@ -0,0 +1,82 @@
+
+:root {
+    --bg-color: #002cbe;
+    --primary-color: #f2f2f2;
+}
+
+#banner { 
+width: 10%;
+position: top;
+
+
+
+}
+.titulo{
+    color: var(--primary-color);
+    font-size: 25px; 
+    text-align: center;
+}
+body {

```

trabalhar padrão no espaçamento

```suggestion

.titulo {
    color: var(--primary-color);
    font-size: 25px; 
    text-align: center;
}

body {

```